pyproject.toml

```diff
@@ -1,3 +1,51 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+[tool.bandit]
+exclude_dirs = ["/venv/"]
+[tool.bandit.assert_used]
+skips = ["*tests/*.py"]
+
+# Testing tools configuration
+[tool.coverage.run]
+branch = true
+
+[tool.coverage.report]
+fail_under = 65
```

Comment:
This is below our minimum, is this expected to be fixed in the future?

Reply:
Yes, this will be part of the rework as a sidecar